editor/src/components/canvas/canvas-strategies/absolute-move-strategy.tsx

```diff
@@ -98,7 +110,10 @@ export function applyAbsoluteMoveCommon(
   canvasState: InteractionCanvasState,
   interactionState: InteractionSession,
   strategyState: StrategyState,
-  getMoveCommands: (snappedDragVector: CanvasPoint) => Array<CanvasCommand>,
+  getMoveCommands: (snappedDragVector: CanvasPoint) => {
```

Comment:
Could we make a type for this return value? As it appears in at least two places fully specified.